storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -746,6 +941,10 @@ Options
    If this option is a relative path, it is interpreted as being relative to
    the current working directory from which :program:`xtrabackup` is executed.
 
+.. option:: --temp-tablespaces-dir=DIRECTORY
+
+   Directory where temp tablespace files live, this path can be absolute.
```

Comment:
this one should probably be "--innodb-temp-tablespaces-dir"... but apparently it has never been used since day 0. will need to rename it or remove

Reply:
fixing by adding innodb